source/js/components/Select/Select.tsx

```diff
@@ -0,0 +1,34 @@
+import { HtmlHTMLAttributes } from 'react';
+
+interface SelectInterface {
+  onChange: React.ChangeEventHandler<HTMLSelectElement>;
+  value: string;
+  label: string;
+  options: { value: string; label: string }[];
+  required?: boolean;
+}
+
+export const Select = ({
+  onChange,
+  id,
+  label,
+  value,
+  options,
+  required = false,
+}: SelectInterface & HtmlHTMLAttributes<HTMLSelectElement>) => (
```

Comment:
Why do we need this:

```js
SelectInterface & HtmlHTMLAttributes<HTMLSelectElement>
```

instead of only this:

```js
SelectInterface
```

Reply:
No need for this now. My thought was that we extend the actual html element we are trying to "componentify" and pass on the attributes.